tar/multitape/multitape_delete.c

```diff
@@ -76,6 +77,7 @@ deletetape(TAPE_D * d, uint64_t machinenum, const char * cachedir,
 	int lockfd;
 	uint8_t lastseq[32];
 	uint8_t seqnum[32];
+	int rc = -1;	/* Assume an error will occur; will change if needed. */
```

Comment:
Hmm.  I would say that setting `rc=-1` is actually "Assume exit is due to non-archive-not-found error", since a successful exit is `rc=0`?  I mean, the initial assumption is that it failed; it changes this assumption only if it succeeds (or has archive-not-found, which is a different failure).